The job **`tidy`** failed! Check out the build log: [(web)](https://github.com/rust-lang/rust/actions/runs/18620449204/job/53090943457) [(plain enhanced)](https://triage.rust-lang.org/gha-logs/rust-lang/rust/53090943457) [(plain)](https://github.com/rust-lang/rust/commit/27a0720a03f731d5ec28b234a7bc7efa6aca851b/checks/53090943457/logs)

<details><summary><i>Click to see the possible cause of the failure (guessed by this bot)</i></summary>

```plain
fmt: checked 6501 files
tidy check
tidy [rustdoc_json (src)]: `rustdoc-json-types` modified, checking format version
tidy: Skipping binary file check, read-only filesystem
tidy [style (src)]: /checkout/src/tools/rust-installer/install-template.sh:554: tab character
tidy [style (src)]: /checkout/src/tools/rust-installer/install-template.sh:562: tab character
tidy [style (src)]: FAIL
removing old virtual environment
creating virtual environment at '/checkout/obj/build/venv' using 'python3.10' and 'venv'
creating virtual environment at '/checkout/obj/build/venv' using 'python3.10' and 'virtualenv'
Requirement already satisfied: pip in ./build/venv/lib/python3.10/site-packages (25.2)
linting python files
---
info: ES-Check: there were no ES version matching errors!  🎉
typechecking javascript files
tidy: The following check failed: style (src)
Bootstrap failed while executing `test src/tools/tidy tidyselftest --extra-checks=py,cpp,js,spellcheck`
Command `/checkout/obj/build/x86_64-unknown-linux-gnu/stage1-tools-bin/rust-tidy /checkout /checkout/obj/build/x86_64-unknown-linux-gnu/stage0/bin/cargo /checkout/obj/build 4 /node/bin/npm --extra-checks=py,cpp,js,spellcheck` failed with exit code 1
Created at: src/bootstrap/src/core/build_steps/tool.rs:1549:23
Executed at: src/bootstrap/src/core/build_steps/test.rs:1280:29

Command has failed. Rerun with -v to see more details.
Build completed unsuccessfully in 0:03:21
  local time: Sat Oct 18 20:21:13 UTC 2025
  network time: Sat, 18 Oct 2025 20:21:13 GMT
##[error]Process completed with exit code 1.
##[group]Run echo "disk usage:"
```

</details>

--- a/src/tools/rust-installer/install-template.sh
+++ b/src/tools/rust-installer/install-template.sh
@@ -551,7 +551,7 @@ install_components() {
         # Decide the destination of the file
         local _file_install_path="$_dest_prefix/$_file"
 
-	local _is_bin=false
+        local _is_bin=false
         case "$_file" in
             etc/*)
                 local _f="$(echo "$_file" | sed 's/^etc\///')"
@@ -559,7 +559,7 @@ install_components() {
                 ;;
             bin/*)
                 local _f="$(echo "$_file" | sed 's/^bin\///')"
-		_is_bin=true
+                _is_bin=true
                 _file_install_path="$CFG_BINDIR/$_f"
                 ;;
             lib/*)

